randovania/games/dread/exporter/patch_data_factory.py

```diff
@@ -338,6 +338,24 @@ def _static_room_name_fixes(self, scenario_name: str, area: Area):
 
         return cc_name, area.name
 
+    def _build_teleporter_name_dict(self, cc_dict: dict = {}) -> dict[str, dict[str, str]]:
```

Comment:
First, don't ever use mutable objects as default arguments. The default argument is shared between all calls and it's not recreated for every call, for fun(tm) bugs.

Second, let's not have this argument.